DTS-STN/secure-client-hub
# Updated modal to use contentLabel depending on context

## [ADO-129445](https://dev.azure.com/VP-BD/DECD/_workitems/edit/129445)

### Description
This PR updates the `MultiModal` component to set the `contentLabel` variable (which React-Modal parses into `aria-label`) based on the modal context. I've also removed the `SessionTimeout` component as it was an unused duplicate/old version of `CountDown`.

### What to test for/How to test
1. Pull in branch
2. Type `npm run dev`
3. Navigate to the dashboard, click one of items in the cards to open the exit beta modal
4. Right click, inspect the element and look for `<div class="ReactModal__Content ReactModal__Content--after-open...`
5. Ensure it has an aria-label
